### PR TITLE
Add support for Django 1.9

### PR DESCRIPTION
**Pull request by wo0dyn**
## ChangeLog

* Add support for Django 1.9
* Remove pattern() in urls.py
* Add Django/Python matrix to tox configuration

### CODE REVIEW

File: formidable/urls.py [hunk 1]
```diff
@@ -1,19 +1,18 @@
-from django.conf.urls import patterns, url
+from django.conf.urls import url
 
-from formidable.views import (
-    AccessList, ContextFormDetail, FormidableCreate, FormidableDetail,
-    PresetsList, ValidateView
-)
+from formidable import views
```
Comment (moumoutte) [inline formidable/urls.py]: Good point, thxs.

Comment (wo0dyn) [inline formidable/urls.py]: :wink:

File: circle.yml [hunk 2]
```diff
@@ -4,5 +4,5 @@ dependencies:
 
 test:
     override:
-        - tox -e py27,py30,py31,py34
+        - tox -r
```
Comment (wo0dyn) [inline circle.yml]: Nope, it's done. I forgot to [fix machine pyenv](https://circleci.com/docs/build-image-precise/#python) into the `circle.yml` file to allow all python versions we need for the tests.
It's green now. :wink:

**Comment by moumoutte** [comment]
Many thxs :)

**Pull request merged**